### ISSUE TITLE
Feature Request: Display branch name behind PR build

### ISSUE THREAD

**Issue by smiles3983** [jenkinsci/build-monitor-plugin#443]
Our project only builds PR's, but I want to see the name of the branch and be able to filter the monitor view by branch name instead of the PR name, which is just a number for us.  Would it be possible to add a switch to "Mask PR names with branch names"?

**Comment by dcendents** [comment]
I have the same issue, but I don't think this plugin can or should attempt to fix this.

What I plan to do, is to set the job display name with the branch name, so this way it doesn't matter what the actual job name is.

Unless the gitlab plugin, which I am using to build merge requests, prevents me from doing this, that should solves the problem.

See: https://support.cloudbees.com/hc/en-us/articles/220860347-How-to-set-build-name-in-Pipeline-job-

**Comment by timja** [comment]
Agreed you should configure your job name to do this. By default its now the PR title instead of the PR number
